locales/en-US/notes.properties

```diff
@@ -1,5 +1,45 @@
-welcomeTitle2=Hello!
-welcomeText2=Welcome to this one-page notepad built in to Firefox. Browse the web. Jot down notes. It’s that simple.
+welcomeTitle3=Welcome to Firefox Notes!
+welcomeSubtitle3=A simple tool for taking notes while you browse.
+welcomeOpenNotes3=To open Notes:
+# LOCALIZATION NOTE (welcomeWindowsLinuxShortcut3): {shortcut} is the
+# key combination to open the sidebar
+welcomeWindowsLinuxShortcut3=On Windows, Linux: {shortcut}
+# LOCALIZATION NOTE (welcomeMacShortcut3): {shortcut} is the
+# key combination to open the sidebar
+welcomeMacShortcut3=On Mac: {shortcut}
+welcomeAccessNotes3=To access your notes from any laptop or desktop computer:
+# LOCALIZATION NOTE (welcomeSyncInfo3): {sync} is the label on the sync button
+welcomeSyncInfo3=Select {sync} to sign into your Firefox Account.
+welcomeFormatText3=<strong>To format your text</strong>, use the toolbar or these commands:
+# LOCALIZATION NOTE (welcomeHeading3): #, ##, and ### are the markdown
+# syntax to use. Keep the backticks to explain to the user how to use
+# the syntax.
+welcomeHeading3=Headings: Begin a line with `#` or `##` or `###` followed by a space to create a heading 1, heading 2, or heading 3
+# LOCALIZATION NOTE (welcomeBold3): **text**, and __text__ are the
+# markdown syntax example. text can be translated. Keep the backticks
+# to explain to the user how to use the syntax.
+welcomeBold3=Bold: `**text**` or `__text__`
+# LOCALIZATION NOTE (welcomeItalics3): *text*, and _text_ are the
+# markdown syntax example. text can be translated. Keep the backticks
+# to explain to the user how to use the syntax.
+welcomeItalics3=Italics: `*text*` or `_text_`
+# LOCALIZATION NOTE (welcomeBulleted3): * and - are the markdown
+# syntax example. Keep the backticks to explain to the user how to use
+# the syntax.
+welcomeBulleted3=Bulleted list: Begin a line with `*`  or  `–`  followed by a space
+# LOCALIZATION NOTE (welcomeNumbered3): 1. and 1) are the markdown
+# syntax example. Keep the backticks to explain to the user how to use
+# the syntax.
+welcomeNumbered3=Numbered list: Begin a line with `1.` or `1)` followed by a space
+# LOCALIZATION NOTE (welcomeCode3): `text` is the markdown syntax
+# example. text can be translated. Please keep the backticks to
+# explain to the user how to use the syntax.
+welcomeCode3=Code: ``text``
+welcomeSuggestion3=Have a suggestion to improve Notes?
+# LOCALIZATION NOTE (welcomeGiveFeedback3): {feedback} contains the
+# label of the Give Feedback button.
+welcomeGiveFeedback3=Select {feedback} in the Notes menu.
+welcomeThatsIt3=<strong>That’s it</strong>—you’re ready to take notes!
```

Comment:
Missing spaces around the dash?

Reply:
In the original version #564 they are not present, I believe it is to have it all on the same line maybe?

Reply:
Ah! It's an em dash
`<strong>That’s it—</strong>`

And apparently it's correct without spaces. English is weird :)
http://www.thepunctuationguide.com/em-dash.html

Reply:
Thank you for checking